setup.py

```diff
@@ -5,11 +5,6 @@
 
 deps = []
 
-# Python 3 unsupported
```

Comment:
We probably only want to remove this when full Python3 support is present, so we don't get bug reports for things we know aren't working yet